platform/broadcom/sai.mk

```diff
@@ -1,12 +1,16 @@
-BRCM_SAI = libsaibcm_6.0.0.10_amd64.deb
-$(BRCM_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/6.0/master/libsaibcm_6.0.0.10_amd64.deb?sv=2020-08-04&st=2021-10-20T15%3A46%3A03Z&se=2030-10-21T15%3A46%3A00Z&sr=b&sp=r&sig=zhosphas9cBuH%2B7mLC%2ByarQPQSIe2LY%2FXOATIW96cVY%3D"
-BRCM_SAI_DEV = libsaibcm-dev_6.0.0.10_amd64.deb
+LIBSAIBCM_VERSION = 4.3.5.1
+LIBSAIBCM_MAIN_VERSION = 4.3
+LIBSAIBCM_BUILD = 48112
```

Comment:
We can remove it, for one package version it should always to the unique commit.